com.tracktion.Waveform13.yml

```diff
@@ -0,0 +1,160 @@
+id: com.tracktion.Waveform13
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform
+finish-args:
+  - --device=all
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=home/.lxvst:create
+  - --filesystem=home/.vst:create
+  - --filesystem=home/.vst3:create
```

Comment:
no do NOT create directories in home.

Reply:
I apologize. This was forked from the original Waveform package and this slipped over my sight.